### PR TITLE
Tag ColorSchemes v1.4.0

### PR DESCRIPTION
**Pull request by cormullion**
I moved this repo to JuliaGraphics and attempted to tag a new release without the help of Attobot. I have a suspicion that I've done something wrong, but haven't yet worked out what it might be...

### CODE REVIEW

File: ColorSchemes/versions/1.4.0/requires [hunk 1]
```diff
@@ -0,0 +1,8 @@
+julia 0.6-
+Images 0.10.0
```
Comment (tkelman) [inline ColorSchemes/versions/1.4.0/requires]: does this need to be tightened? the `reshape` with colon wouldn't work on early 0.6-dev versions so better to use 0.6 release as the minimum julia version

Comment (cormullion) [inline ColorSchemes/versions/1.4.0/requires]: Good thinking. Will change it.

Comment (tkelman) [inline ColorSchemes/versions/1.4.0/requires]: and what about Images, are you relying on a feature or bugfix that was in 0.10 but not 0.6?

Comment (cormullion) [inline ColorSchemes/versions/1.4.0/requires]: Oh, I'll have to research that. I just got it working with the current version of Images and updated the versions. Images moves fast....

**Comment by tkelman** [comment]
replaced by https://github.com/JuliaLang/METADATA.jl/pull/10325 ?

**Comment by cormullion** [comment]
I suppose I could delete this one?

**Comment by tkelman** [comment]
if redundant, yeah

**Comment by cormullion** [comment]
Moving repos is a bit tricky, IMHO.

**Pull request closed without merging**